doc/reference/cmdline.rst

```diff
@@ -38,9 +53,28 @@ Running the server
 
     .. (nb: when and why?)
 
+.. option:: --upgrade-path <upgrade_path>
+
+   specify an additional upgrade path.
+
+.. option:: --load <modules>
+
+   comma-separated list of server-wide modules. Default is ``base,web``.
+
 .. option:: -c <config>, --config <config>
 
-    provide an alternate :ref:`configuration file <reference/cmdline/config>`
+    path to an alternate :ref:`configuration file <reference/cmdline/config>`.
+    If not defined, Odoo checks ``ODOO_RC`` environmental variable
+    and default location :file:`{$HOME}/.odoorc`.
+    See configuration file section :ref:`below <reference/cmdline/server/config>`.
+
+.. option:: -D <data-dir-path>, --data-dir <data-dir-path>
+
+   directory path where to store Odoo data (eg. filestore, sessions).
+   If not specified, Odoo will fallback
+   to a predefined path. On Unix systems its
+   one defined in ``$XDG_DATA_HOME`` environmental variable
+   or :file:`~/.local/share/Odoo` or :file:`/var/lib/Odoo`.
```

Comment:
Seems more confusing than helpful. And `~/.local/share` is the prescribed fallback for `XDG_DATA_HOME`.

Also not "Unix systems". macOS is very much a UNIX system, but there Odoo uses the prescribed `~/Library/Application Support`

> or :file:`/var/lib/Odoo`.

That is for the specific case where Odoo is not run as a user (`~` does not expand to a directory). And in that situation `Odoo` is specifically lowercased.

Reply:
information was taken from odoo/odoo/tools/appdirs.py

My goal here is to help users find default paths, as they are not documented in official docs and there are plenty of questions regarding it on odoo forum and stack overflow.

So I see it very important to show possibilities and configuration options to make developers and devops people life easier. :)

Please advice how to proceed:
a) add more comprehensive information about paths (including linux, macos, windows, similar to one in appdirs.py
b) just rephrase the fragment to add information about /var/lib/ special case and change "Unix" to "Linux".

Reply:
Even if it doesn't cover 100% cases, it's already a nice improvement. Let's keep it like this at the moment.